components/ledger/openapi.yaml

```diff
@@ -542,7 +542,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/TransactionResponse'
+                $ref: '#/components/schemas/ReadTransactionResponse'
```

Comment:
I would have named this GetTransactionResponse, what do you think ?

Reply:
Yes I can.
Usually, I like to use Create Read Update Delete names.
It's avoid ambiguity in my head.